### ISSUE TITLE
ID third_party_check: 
User feedback: something
(`third_party_check`)

### ISSUE THREAD

**Issue by MPLP-Docassemble** [mplp/docassemble-MLHMotionRegardingChangeOfDomicile#45]
&nbsp; | &nbsp;
-------|------------------------------------
Question ID | `third_party_check`
Details | I think it is best practice to avoid "no label" fields for accessibility reasons (https://docassemble.org/docs/fields.html#no%20label). Can you add a label like " Is there a third party in your case?"
Variable being sought | `third_party_check`
Package version | `0.0.6`
Filename | `docassemble.MLHMotionRegardingChangeOfDomicile:data/questions/motion_regarding_change_of_domicile.yml`

**Comment by h-mcguire** [comment]
Noted! I'm assuming the accessibility problems happen with screen readers? Updated in commit 996f982

**Comment by ekressmiller** [comment]
That's right! We talked about the complexities of this some more at the dev meeting last week. For some more discussion and examples, please [see the text flagged by this comment.](https://docs.google.com/document/d/1vyPWcC8dUyvJ7hTe7f7M_GBYDKDs3_wqwO-H7ue6l3U/edit?disco=AAABNL11MzU)
